Sources/AccessibilitySnapshot/Core/Swift/Classes/UIAccessibility+SnapshotAdditions.swift

```diff
@@ -512,6 +553,11 @@ extension UIAccessibilityTraits {
 
     static let switchButton = UIAccessibilityTraits(rawValue: 0x0020000000000000)
 
+    static let isEditing = UIAccessibilityTraits(rawValue: 0x0000000000200000)
```

Comment:
I'm assuming this is when the text field is first responder? Does the accessibility description include anything for this trait?

Reply:
Yes, when the text field is the first responder VoiceOver reads "Text field, Is editing, Placeholder text, insertion point at start, Use the rotor to access Misspelled Words"

Reply:
I've made updates to handle this trait.